frontend/styles/ubf.scss

```diff
@@ -248,6 +248,84 @@ $buttons: (
   width: auto;
 }
 
+.ubf-section {
+  // Remove existing margins from textual elements.
+  p, h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+  }
+
+  display: flex;
+  flex-direction: column;
+  padding: $spacer * 3 $spacer * 1.25;
+
+  @include media-breakpoint-up(md) {
+    padding: $spacer * 3.5 $spacer * 2.5;
+  }
+
+  @include media-breakpoint-up(xl) {
+    padding: $spacer * 5 $spacer * 4;
+  }
+
+  @include media-breakpoint-up(xxl) {
+    padding: $spacer * 7 $spacer * 4;
+  }
+}
+
+.faq-section {
+  @extend .ubf-section;
```

Comment:
Instead of using `@extend`, which creates duplicate code, it's usually better to simply use multiple CSS classes in the HTML (i.e. `class="ubf-section faq-section"`).